tests/util.py

```diff
@@ -17,14 +22,83 @@
     PublicFormat,
 )
 
-from jwt_authorizer import config
 from jwt_authorizer.app import create_app
+from jwt_authorizer.config import ALGORITHM
+from jwt_authorizer.verify import KeyClient
 
 if TYPE_CHECKING:
-    from flask import Flask
     from typing import Any, Dict, List, Optional
 
 
+def number_to_base64(data: int) -> bytes:
+    """Convert an integer to base64-encoded bytes in big endian order.
+
+    Parameters
+    ----------
+    data : `int`
+        Arbitrarily large number
+
+    Returns
+    -------
+    result : `bytes`
+        The equivalent URL-safe base64-encoded string corresponding to the
+        number in big endian order.
+    """
+    bit_length = data.bit_length()
+    byte_length = bit_length // 8 + 1
+    data_as_bytes = data.to_bytes(byte_length, byteorder="big", signed=False)
```

Comment:
Never thought I'd have to think about endian issues in python, but it's good to know. :)